### ISSUE TITLE
BUG-008 - WAVE: Icons' alt text is the filepath

### ISSUE THREAD

**Issue by JosephBanham** [lunarcroissant/react-challenge-1-window-news#8]
As a best practice, alt text should be descriptive. Using the filepath is unfortunately not descriptive in the 'correct way' for a blind user to understand what that image or icon should be.

**Comment by lunarcroissant** [comment]
This has now been fixed :)
